migration-tools/migration/globals.py

```diff
@@ -39,7 +39,7 @@
 
 ## DocuBlocks
 ALL_COMMENTS_FILE = f"{OLD_TOOLCHAIN}/{version}/generated/allComments.txt"
-OAPI_COMPONENTS_FILE = f"{args.dst}/migration-tools/arangoproxy/cmd/openapi/components.yaml"
+OAPI_COMPONENTS_FILE = f"{args.dst}/openapi/components.yaml"
```

Comment:
With #1, it's in this place:
```suggestion
OAPI_COMPONENTS_FILE = f"{args.dst}/toolchain/openapi/components.yaml"
```